docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -1,15 +1,190 @@
 [role="xpack"]
+[testenv="basic"]
 [[getting-started-index-lifecycle-management]]
 == Getting started with {ilm}
 
-Create a policy that rolls over after 1 day deletes an index after 30 days
+Let's jump into {ILM} by working through a hands-on scenario.
+This section will leverage many new concepts unique to {ILM} that
+you may not be familiar with. The following sections will explore
+these in more details.
 
-Show create policy API req/res
+The goal of this example is to set up a set of indices that will encapsulate
+the logs from our logging source. We can imagine there is a system
+like {filebeat-ref}[Filebeat] that continuously indexes documents into
+our writing index. We wish to roll over the index after it reaches a size of 50 gigabytes,
+or has been created 30 days ago, and then delete the index after 90 days.
 
-Show assign policy to index API req/res
+=== Setting up a new policy
 
-Show both the API and how it is done with `index.lifecyce.name` using the
-create-index API
+There are many new APIs introduced by {ILM}, but we will only focus on
+a few that are needed for our example. For starters, we will use the
+<<ilm-put-lifecycle,Put Policy>> API to define our first policy. Lifecycle
+policies are defined in JSON and include specific <<ilm-policy-definition,phases and actions>>.
 
-Show explain API to show current state, but ignore the “step” related info,
-only focus on managed/phase/action
+[source,js]
+------------------------
+PUT _ilm/policy/datastream_policy
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "actions": {
+          "rollover": {
+            "max_size": "50GB",
+            "max_age": "30d"
+          }
+        }
+      },
+      "delete": {
+        "min_age": "90d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+------------------------
+// CONSOLE
+// TEST
+
+Here we created the policy called `datastream_policy` which rolls over the index
+being written to after it reaches 50 gigabytes, or it is 30 days old. The rollover will
+occur when any of these conditions is true. After 90 days, our index will be deleted.
```

Comment:
```suggestion
occur when either of these conditions is true. After 90 days, our index will be deleted.
```

Reply:
We should be super clear about when the 90 days starts - I can see some people thinking that it's 90 days after rollover, and some thinking that it's 90 days after the index is created.